modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -68,106 +94,153 @@ mirror:
+====
 
 .Example `ImageSetConfiguration` file
 [source,yaml]
 ----
 apiVersion: mirror.openshift.io/v1alpha2
 kind: ImageSetConfiguration
 storageConfig:
- registry:
-   imageURL: example.com/mirror/oc-mirror-metadata
-   skipTLS: false
+  registry:
+    imageURL: example.com/mirror/oc-mirror-metadata
+    skipTLS: false
 mirror:
-  platform:
-    channels:
-      - name: stable-4.10
-        minVersion: 4.10.10
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.13
+    packages:
+    - name: elasticsearch-operator
+      channels:
+      - name: stable-5.7
+      - name: stable
 ----
 
+// New example; Entire catalog; all versions
 [discrete]
-[id="oc-mirror-image-set-examples-operator-min-max_{context}"]
-== Use case: Including Operator versions from a minimum to a maximum
+[id="oc-mirror-image-set-examples-entire-catalog-full_{context}"]
+== Use case: Including an entire catalog (all versions)
 
-The following `ImageSetConfiguration` file uses a local storage backend and includes only an example Operator, versions starting at `1.0.0` through `2.0.0` in the `stable` channel.
+The following `ImageSetConfiguration` file sets the `mirror.operators.full` field to `true` to include all versions for an entire Operator catalog.
 
-This allows you to only mirror a specific version range of a particular Operator. As time progresses, you can use these settings to adjust the version to newer releases, for example when you no longer have version `1.0.0` running anywhere anymore. In this scenario, you can increase the `minVersion` to something newer, for example `1.5.0`. When oc-mirror runs again with the updated version range, it automatically detects that any releases older than `1.5.0` are no longer required and deletes those from the registry to conserve storage space.
+.Example `ImageSetConfiguration` file
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  registry:
+    imageURL: example.com/mirror/oc-mirror-metadata
+    skipTLS: false
+mirror:
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.13
+      full: true
+----
+
+// New example; Entire catalog; heads only
+// - Included 'targetCatalog' in example
+[discrete]
+[id="oc-mirror-image-set-examples-entire-catalog-heads_{context}"]
+== Use case: Including an entire catalog (channel heads only)
+
+The following `ImageSetConfiguration` file includes the channels heads for an entire Operator catalog.
+
+By default, for each Operator in the catalog, oc-mirror includes the latest Operator version (channel head) from the default channel. If you want to mirror all Operator versions, and not just the channel heads, you must set the `mirror.operators.full` field to `true`.
+
+This example also uses the `targetCatalog` field to specify an alternative namespace and name to mirror the catalog as.
```

Comment:
This wording is a little awkward; maybe "as" can be removed, if it wouldn't change the meaning?

Reply:
I see what you mean, but I think it would change the meaning to remove it. This wording is used elsewhere in the docs already. I'll make a note to see if I can find a better wording and if I can, I'll update everywhere.